src/parser/CMakeLists.txt

```diff
@@ -16,7 +16,6 @@ add_library(parser
         SparqlLexer.h SparqlLexer.cpp
         TokenizerCtre.h TurtleTokenId.h
         ParallelBuffer.cpp
-        SparqlParserHelpers.h SparqlParserHelpers.cpp)
+        SparqlParserHelpers.h SparqlParserHelpers.cpp
+        ../util/GeoSparqlHelpers.h ../util/GeoSparqlHelpers.cpp)
```

Comment:
Please Create a `util` library in the `util` folder, if we start compiling .cpp files very far away from their folder, we will never find them again.

Reply:
I see, you have done this. So then the solution should be to add `util` in the `target_link_libraries` command below.

Reply:
Done!